csource/csource_test.go

```diff
@@ -32,20 +32,33 @@ func allOptionsPermutations() []Options {
 	for _, opt.Threaded = range []bool{false, true} {
 		for _, opt.Collide = range []bool{false, true} {
 			for _, opt.Repeat = range []bool{false, true} {
-				for _, opt.Repro = range []bool{false, true} {
-					for _, opt.Procs = range []int{1, 4} {
-						for _, opt.Sandbox = range []string{"none", "setuid", "namespace"} {
+				for _, opt.Procs = range []int{1, 4} {
```

Comment:
This not takes too long even in short mode, even with high parallelism.
Let's test options independently in short mode (i.e. 1 run for threaded, 1 run for handlesegv, etc). And all permutations in non-short mode.